src/api/osm.ts

```diff
@@ -37,4 +37,14 @@ export async function fetchOsmWaysForNode(nodeId: string | number): Promise<Way[
     }))
     .filter((way: Way) => Object.keys(way.tags).length !== 0);
   return ways;
+}
+
+export async function isNodeValid(osmNode: string | number) {
```

Comment:
Could you also add some basic checks that the `osmNode` is also a valid integer number? This would mean we fail faster/minimise invalid requests to the OSM API (though this is a nice to have, happy to merge without!)

Reply:
Yep, I have added a check using regex that ensures there are only numbers in the nodeId, I have also had a look at the max value of OSM nodeId and put an upper and lower limit. I am a bit hesitant to have an upper limit because it will change even though it works now.